pypit/ararc.py

```diff
@@ -236,34 +238,36 @@ def setup_param(slf, sc, det, fitsdict):
             arcparam['disp'] = 1.43
         else:
             msgs.error('Not ready for this disperser {:s}!'.format(disperser))
-    elif sname=='lris_red':
+    elif sname=='keck_lris_red':
         arcparam['wv_cen'] = fitsdict['headers'][idx[0]][0]['WAVELEN']
         lamps = ['ArI','NeI','HgI','KrI','XeI']  # Should set according to the lamps that were on
         if disperser == '600/7500':
-            arcparam['n_first']=3 # Too much curvature for 1st order
+            arcparam['n_first']=2 # Too much curvature for 1st order
             arcparam['disp']=0.80 # Ang per pixel (unbinned)
-            arcparam['b1']= 1./arcparam['disp']/slf._msarc[det-1].shape[0]
+            arcparam['disp_toler']=0.05 # Tolerance of dispersion
+            arcparam['b1']= 1./arcparam['disp']/slf._msarc[det-1].shape[0] / binspectral
             arcparam['wvmnx'][1] = 11000.
+            arcparam['Nstrong'] = 30
```

Comment:
This could be trouble for some of the lower resolution gratings..

Reply:
Ah, I see it is grating dependent.  Could still be trouble.  ;)

Reply:
OK, I'll change this back. It's not used with the new arclines code. I think I did this while I was trying to "fix" the solution for LRISr.